ambry-api/src/main/java/com/github/ambry/accountstats/AccountStatsStore.java

```diff
@@ -84,6 +86,14 @@ public interface AccountStatsStore {
    */
   AggregatedAccountStorageStats queryMonthlyAggregatedAccountStorageStats() throws Exception;
 
+  /**
+   * Retain the host account storage stats for the given list of host and port, and remove the account storage stats for
+   * the other hosts.
+   * @param hosts The list of host and port pair
+   * @throws Exception
+   */
+  void retainHostAccountStorageStatsForHosts(List<Pair<String, Integer>> hosts) throws Exception;
```

Comment:
nit: Since we want to remove old hosts from mysql, would name like `deleteAccountStorageStatesForInvalidHosts` be better?